packages/wallet/src/services/UtxoTracker.ts

```diff
@@ -17,6 +18,7 @@ export interface UtxoTrackerProps {
   addresses$: Observable<Cardano.PaymentAddress[]>;
   stores: Pick<WalletStores, 'utxo' | 'unspendableUtxo'>;
   transactionsInFlight$: Observable<TxInFlight[]>;
+  signedTransactions$: Observable<SignedTx[]>;
```

Comment:
Since we won't be using explicit dismissal (see LW-9558), we need inputs in signed$ to be available for spending. Otherwise, wallet could get into a state where it's locked forever (unsubmitted, unspendable utxo and no validity interval). Please undo all changes in UtxoTracker, and add new logic that uses signed$ to WalletUtil input resolver
```suggestion
```

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/1113/commits/07eb21562c7f0476b6d429b9d6693f43f8a199a0

Reply:
Input resolver: https://github.com/input-output-hk/cardano-js-sdk/pull/1113/commits/1f172701553e0db90e76f2dcd628c54b2ec8060d